### ISSUE TITLE
cannot find package "github.com/prometheus/client_golang/text"

### ISSUE THREAD

**Issue by brunoqc** [prometheus/prometheus#1165]
I'm trying to install using the instructions on https://github.com/prometheus/prometheus#building-from-source

```
$ env GOPATH=/opt/prometheus go get github.com/prometheus/prometheus/...
package github.com/prometheus/prometheus/vendor/github.com/golang/snappy: code in directory /opt/prometheus/src/github.com/prometheus/prometheus/vendor/github.com/golang/snappy expects import "github.com/golang/snappy"
package github.com/prometheus/prometheus/vendor/golang.org/x/net/context: code in directory /opt/prometheus/src/github.com/prometheus/prometheus/vendor/golang.org/x/net/context expects import "golang.org/x/net/context"
package github.com/prometheus/prometheus/vendor/github.com/prometheus/client_golang/prometheus
        imports github.com/prometheus/client_golang/text: cannot find package "github.com/prometheus/client_golang/text" in any of:
        /usr/local/go/src/github.com/prometheus/client_golang/text (from $GOROOT)
        /opt/prometheus/src/github.com/prometheus/client_golang/text (from $GOPATH)
```

go version go1.5.1 linux/386

**Comment by juliusv** [comment]
@brunoqc Thanks for reporting this! Indeed, we changed the way we do vendoring, which needs the `GO15VENDOREXPERIMENT=1` env var to be set. And it would also be better to change it to `go get github.com/prometheus/prometheus/cmd/...` because that would not have tried to directly build the packages under `./vendor` then, so the build would still have succeeded, albeit not using the vendored library versions.

I filed a PR for that: https://github.com/prometheus/prometheus/pull/1166
